app/src/organisms/ChangePipette/index.tsx

```diff
@@ -37,10 +37,11 @@ import { ClearDeckModal } from './ClearDeckModal/index'
 import { ExitAlertModal } from './ExitAlertModal'
 import { DeprecatedInstructions } from './DeprecatedInstructions'
 import { ConfirmPipette } from './ConfirmPipette'
+import { DeprecatedConfirmPipette } from './DeprecatedConfirmPipette'
 import { RequestInProgressModal } from './RequestInProgressModal'
-import { LevelPipette } from './LevelPipette'
 import { DeprecatedLevelPipette } from './DeprecatedLevelPipette'
 import { ClearDeckAlertModal } from './ClearDeckModal/ClearDeckAlertModal'
+import { Mount, SPACING } from '@opentrons/components'
```

Comment:
Add the `type` lower with the other `types`

```suggestion
import { SPACING } from '@opentrons/components'
import type { Mount } from '@opentrons/components'
```

Reply:
Also wait, is that the right import for `Mount`, Shouldn't it be imported from `redux/pipettes/types`?

Reply:
They both seem to be same exact thing. I will switch the import to the redux directory though.